packages/block-library/src/calendar/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'core/calendar';
+
+export const settings = {
+	title: __( 'Calendar' ),
+
+	description: __( 'A calendar of your site’s Posts.' ),
```

Comment:
Let's lowercase 'posts' here since it's not a proper noun or a title-case sentence.

```suggestion
	description: __( 'A calendar of your site’s posts.' ),
```